bidwire/alembic/versions/a1b42c9006a7_absolute_massgov_eopss_url.py

```diff
@@ -17,7 +17,7 @@
 
 # revision identifiers, used by Alembic.
 revision = 'a1b42c9006a7'
-down_revision = '9b30b0fe231a'
```

Comment:
You should not be changing an existing and already-applied database migration. Please revert changes to this file and make your revision follow this one, not the other way around.